src/grt/src/GlobalRouter.tcl

```diff
@@ -387,5 +396,14 @@ proc define_clock_layer_range { layers } {
   }
 }
 
+proc check_pdrev_alpha { alpha } {
```

Comment:
bad name; this does not check alpha, it sets it. not usable to check alpha for the other command so the name is misleading